lib/runtime/src/component.rs

```diff
@@ -329,6 +349,18 @@ impl MetricsRegistry for Endpoint {
         ]
         .concat()
     }
+
+    fn stored_labels(&self) -> Vec<(&str, &str)> {
+        // Convert Vec<(String, String)> to Vec<(&str, &str)>
+        self.labels
+            .iter()
+            .map(|(k, v)| (k.as_str(), v.as_str()))
+            .collect()
+    }
+
+    fn labels_mut(&mut self) -> &mut Vec<(String, String)> {
+        &mut self.labels
+    }
```

Comment:
The same methods appear at line 200. The diff is making it tricky for me to see what they are both part of. Are these duplicates?